# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp
# Commit: af562cec3d70
# Full Hash: af562cec3d706ee3ed605ffeade4d3ba982d2545
# Author: Ray Kraesig <rkraesig@mozilla.com>
# Date: 2023-10-27 04:08:13
# Regressor Bug: 1837079
# File Overlap Count: 2
# Description:
#   Bug 1837079 - [5/10] Create WinFileDialog IPDL protocol and implementation  r=gstoll,handyman,ipc-reviewers,nika,win-reviewers,mhowell
#   
#   Create and implement a new top-level IPC protocol, `PWinFileDialog`,
#   using the primitives from bug 1833450 and the new sandboxing type from
#   the previous commit.
# ==============================================================================

diff -r b5d52e5f75f2 -r af562cec3d70 ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp
--- a/ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp	Thu Oct 26 18:21:28 2023 +0000
+++ b/ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp	Thu Oct 26 18:21:28 2023 +0000
@@ -5,8 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #if defined(ENABLE_TESTS)
+#  include "mozilla/ipc/UtilityProcessManager.h"
 #  include "mozilla/ipc/UtilityProcessTest.h"
-#  include "mozilla/ipc/UtilityProcessManager.h"
 #  include "mozilla/dom/Promise.h"
 #  include "mozilla/ProcInfo.h"
 #  include "mozilla/IntentionalCrash.h"